lualib/resty/events/frame.lua

```diff
@@ -82,12 +89,17 @@ function _M.send(sock, payload)
         return nil, err
     end
 
-    local bytes, err = sock:send(uint16_to_bytes(payload_len) .. payload)
+    local data = SEND_DATA
```

Comment:
Here I used `tablepool` but remove it now.

I think that we won't get race condition, am I right?

Reply:
Can we just not to use the upvalue as `local data = SEND_DAT` and make the data structure to be sent directly, like:

```
local bytes, err = sock:send({ uint_to_bytes(payload_len), payload })
```

I'm afraid I don't understand why you tend to use the upvalue here.

Reply:
It will create too many temporary small tables, increase the burden of gc.